test/metrics_unittest.cc

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+#include "../../src/metrics.h"
+#include "gtest/gtest.h"
+
+namespace google {
+
+TEST(SerializeToPrometheusTextTest, Initialization) {
+  EXPECT_EQ("",
```

Comment:
Also call `RegisterAllViewsForExport()`?

Reply:
Based on the comment on [having metrics/view registration in one function, and Igor and I explored it on Friday afternoon, current codebase doesn't have this function at the moment.